### DISCUSSION TITLE
How to define 9-bit SWSPI for use with Arduino nano and ILI9341

### DISCUSSION THREAD

**Discussion by erikan88** [moononournation/Arduino_GFX#286] [Q&A] [answered]
When reading trough the documentation, I am under the impression that it is possible to run 9-bit SWSPI (altough slow) on an Arduino Nano connected to a display with ILI9341 driver. 
Is this possible? If so- how do I define Arduino_SWSPI to use 9-bit instead of 8-bit?

Thanks

**Comment by moononournation** [comment]
no DC pin will be 9-bit

**Reply by erikan88** [reply]
Do I understand correctly if removing "9 /* DC */" from code below is the same as declarning no DC pin ?

Arduino_DataBus *bus = new Arduino_SWSPI(9 /* DC */, 10 /* CS */, 13 /* SCK */, 11 /* MOSI */, GFX_NOT_DEFINED /* MISO */);
-->
Arduino_DataBus *bus = new Arduino_SWSPI(10 /* CS */, 13 /* SCK */, 11 /* MOSI */, 12 /* MISO */);

My schematic below: 

The Mode that is used on the display assembly is this one:
![image](https://user-images.githubusercontent.com/58425010/233578764-04f2a312-1a1d-42f7-8b76-fac287c2b9b7.png)

**Reply by erikan88** [reply]
![image](https://user-images.githubusercontent.com/58425010/233579793-b5a0f086-bc19-4ffd-8c39-3183e443adc7.png)

**Reply by moononournation** [reply, marked as answer]
you should pass GFX_NOT_DEFINED as DC parameter

**Reply by erikan88** [reply]
That did the trick, it is working as intended now!
Thank you for your quick support :)
